features/google/src/model/importers/semantic-locations-importer.js

```diff
@@ -0,0 +1,89 @@
+import ActivitySegment from "../entities/activity-segment";
+import PlaceVisit from "../entities/place-visit";
+
+const semanticLocationsRegex =
+    /\/Location History\/Semantic Location History\/.*\.json$/;
```

Comment:
My Spanish takeout includes "Historial de ubicaciones", then many yearly subdirectories, and right, 2022_APRIL.json files. This is also more of a glob pattern than a regex. My point anyway is that we should try and find another way of locating these location history.
In the same way that we looked for common directories and uncommon ones in Facebook by looking at different examples, we should do the same here, looking at different takeout zip files and then finding strategies to locate the information.
This is just an example, BTW. Others might be totally different.

Reply:
In this case I was just targeting English; but in this particular case to get this forward we could simply check for `Semantic Location History`.

Reply:
Please bear in mind that, in my case, there are yearly directories in that one. Maybe the regex could be restricted a bit more by using /\d+/\w+\.json, or even the name of the file itself.

Reply:
I'm using now `/\/[^/]+\/Semantic Location History\/.*\.json$/`. Feels good enough for the moment. If we get some file that we cannot parse the importer will fail.

Reply:
I updated it to `/\/[^/]+\/Semantic Location History\/\d+\/[^.]+\.json$/` to take also the year as a folder into acount.